http3/response_writer.go

```diff
@@ -50,6 +51,41 @@ func (w *responseWriter) Header() http.Header {
 	return w.header
 }
 
+// flushHeader send header frame in wire format when:
```

Comment:
I don't understand this comment.

Reply:
This comment means when the header will be flushed to the client (i.e. encoded as valid quic datagrams and sent).